warehouse/legacy/api/json.py

```diff
@@ -173,6 +173,7 @@ def _json_data(request, project, release, *, all_releases):
 
     if all_releases:
         data["releases"] = releases
+        data["versions"] = sorted(releases)
```

Comment:
I guess a question here is whether we want to align the version-less and the versioned urls and include the `versions` key on all responses like we used to include the `releases` key.

Reply:
The way I'm thinking about it is that the version-less endpoint helps you to find details for particular versions (and also provides details for whatever the latest version is).  Then the versioned endpoints just give versioned information.

Certainly poetry doesn't have a use case for getting the full version list from a versioned endpoint, so I'm comfortable leaving it off.

Say if you want it in, presumably it's just a case of unindenting this line...

Reply:
I reckon it only makes sense for the unversioned URL.